application/binary/build.gradle.kts

```diff
@@ -2,26 +2,27 @@ androidBinary(
     packageName = "com.stepango.blockme.app",
     owner = Teams.core,
     dependencies = deps(
-        project(":root-app"),
+        target(":root-app"),
 
-        project(":feature:home:api"),
-        project(":feature:home:impl"),
-        project(":feature:home:databinding"),
-        project(":feature:characters:core:api"),
-        project(":feature:characters:core:impl"),
-        project(":feature:characters:list:api"),
-        project(":feature:characters:list:impl"),
-        project(":feature:characters:list:databinding"),
-        project(":feature:characters:detail:api"),
-        project(":feature:characters:detail:impl"),
-        project(":feature:characters:detail:databinding"),
-        project(":feature:characters:favorite:api"),
-        project(":feature:characters:favorite:impl"),
-        project(":feature:characters:favorite:databinding"),
+        target(":feature:home:api"),
+        target(":feature:home:impl"),
+        target(":feature:home:databinding"),
+        target(":feature:characters:core:api"),
+        target(":feature:characters:core:impl"),
+        target(":feature:characters:list:api"),
+        target(":feature:characters:list:impl"),
+        target(":feature:characters:list:databinding"),
+        target(":feature:characters:detail:api"),
+        target(":feature:characters:detail:impl"),
+        target(":feature:characters:detail:databinding"),
+        target(":feature:characters:favorite:api"),
+        target(":feature:characters:favorite:impl"),
+        target(":feature:characters:favorite:databinding"),
 
-        project(":common:extensions:android-util"),
-        project(":common:progressbar:databinding"),
-        project(":core:mvvm:library"),
-        project(":core:di:library")
+        target(":common:extensions:android-util"),
+        target(":common:progressbar:databinding"),
+        target(":common:util-native"),
```

Comment:
I think, name for target should be `:common:native-util` as other targets  `android-util`, `test-util` and etc.
 It's better for clear project structure and consistence.

Reply:
I guess `-native` is a suffix here, but the target itself should not be named as `util`

Reply:
That right, `-native` is a suffix. But I could rename `utils` to `secure` or smth like this.

Reply:
No need. I thought you did utils behavior, but here is only [any name]-native